README.md

```diff
@@ -122,20 +137,20 @@ If you plan to develop with TorchServe and change some of the source code, you m
 1. Make your changes executable
 
     ```bash
-    pip install -e .
+    pip install .
     ```
 
 * To develop with torch-model-archiver:
 
 ```bash
 cd serve/model-archiver
-pip install -e .
+pip install .
```

Comment:
The instruction is to make it executable, but you remove the executable flag.

Reply:
Please refer the comment above.